File: w09/d01/james_rothpearl/mail_items/spec/mailitem_spec.rb
require_relative 'spec_helper'
require_relative '../lib/mailitem'

describe Mailitem do
  it "has a shipping address" do
    m1 = Mailitem.new(ship_to_address: "902 Broadway")
    expect(m1.ship_to_address).to eq("902 Broadway")
  end

  it "should have a stamp value" do
    m1 = Mailitem.new(ship_to_address: "902 Broadway", stamp_value: 42)
    expect(m1.stamp_value).to eq(42)
  end
end

File: w09/d03/will_smith/mail_objects_app/spec/javascripts/support/mailitem_spec.js
describe("a mailitem", function(){
  var mailitem;
  beforeEach(function(){
    mailitem = new Mailitem();
    mailitem.shipToAddress = "118 East 1st Street";
    mailitem.stampValue = 42;
  });

  it ("has a shipping address", function(){
    expect(mailitem.shipToAddress).toEqual("118 East 1st Street");
  });

  it ("has a stamp that has a value", function(){
    expect(mailitem.stampValue).toEqual(42);
  });
});
